src/main/webapp/resources/js/apis/pipelines/pipelines.js

```diff
@@ -27,10 +28,70 @@ export async function getIridaWorkflowDescription(workflowUUID) {
  * Get a listing of all Pipelines in IRIDA.
  * @returns {Promise<AxiosResponse<any> | never>}
  */
-export const fetchIridaAnalysisWorkflows = async function() {
-  var ajaxUrl = URL;
-  if (window.PAGE.automatedProject !== null) {
-    ajaxUrl = `${ajaxUrl}?automatedProject=${window.PAGE.automatedProject}`;
-  }
-  return axios.get(ajaxUrl).then(response => response.data);
+export const fetchIridaAnalysisWorkflows = async function () {
+  return axios.get(AJAX_URL).then((response) => response.data);
+};
+
+export const fetchAutomatedIridaAnalysisWorkflows = async function () {
```

Comment:
Add function comment

Reply:
Updated in 4aae718c0453a86ea3a81b9e005e1d003d9ca5be